javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javassistmodel/JavassistInterfaceDeclaration.java

```diff
@@ -182,19 +182,7 @@ public boolean isAssignableBy(ResolvedReferenceTypeDeclaration other) {
 
     @Override
     public boolean hasDirectlyAnnotation(String canonicalName) {
-        try {
-            for (Object annotationRaw : ctClass.getAnnotations()) {
-                if (annotationRaw.getClass().getCanonicalName().equals(canonicalName)) {
-                    return true;
-                }
-                if (Arrays.stream(annotationRaw.getClass().getInterfaces()).anyMatch(it -> it.getCanonicalName().equals(canonicalName))) {
-                    return true;
-                }
-            }
-        } catch (ClassNotFoundException e) {
-            throw new RuntimeException(e);
-        }
-        return false;
+        return ctClass.hasAnnotation(canonicalName);
```

Comment:
I'd like to see what @ftomassetti thinks of this before I merge.

Reply:
It seems correct to me. My only doubt is it this try tro load the class or not, I think not, so it should be ok